rcl/test/rcl/test_arguments.cpp

```diff
@@ -235,11 +237,24 @@ TEST_F(CLASSNAME(TestArgumentsFixture, RMW_IMPLEMENTATION), check_valid_vs_inval
   EXPECT_FALSE(are_valid_ros_args({"--ros-args", "-p"}));
   EXPECT_FALSE(are_valid_ros_args({"--ros-args", "--params-file"}));
 
-  EXPECT_FALSE(are_valid_ros_args({"--ros-args", "-p", ":="}));
-  EXPECT_FALSE(are_valid_ros_args({"--ros-args", "-p", "foo:="}));
-  EXPECT_FALSE(are_valid_ros_args({"--ros-args", "-p", ":=bar"}));
-  EXPECT_FALSE(are_valid_ros_args({"--ros-args", "-r", "__ns:="}));
```

Comment:
Did those stopped being valid?

Reply:
Nope! Didn't mean to remove them. See https://github.com/ros2/rcl/pull/777/commits/85c74458166aab2e3eb5d23cebac52b6fcf38308. Re-ordered things a bit too.